src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -86,20 +100,18 @@ public GetCredentialCommand() : base()
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!string.IsNullOrEmpty(this.Message))
-            {
-                string caption = UtilsStrings.PromptForCredential_DefaultCaption;
-                string message = this.Message;
+            string caption = this.Title ?? UtilsStrings.PromptForCredential_DefaultCaption;
+            string message = this.Message ?? "Enter your credentials.";
```

Comment:
you can add the default message to `Microsoft.PowerShell.Security\resources\UtilsStrings.resx`. You can find the `PromptForCredential_DefaultCaption` defined there.